--- ./src/interfaces/IAccessControlFacet.sol
+++ ./src/interfaces/IAccessControlFacet.sol
@@ -23,7 +23,20 @@
     /// @dev Set to `address(0)` to renounce any ownership.
     /// @param owner_ The address of the new owner of the contract
     function transferOwnership(address owner_) external;
+    /**
+     * @notice Sets the StarkEx operator
+     * @dev The StarkEx operator is the address that has the authority to call certain
+     *  functions that affect the entire StarkEx system.
+     * @param operator_ The address of the StarkEx operator
+     * @return None
+     */
     function setStarkExOperator(address operator_) external;
+    /**
+     * @notice Sets the interoperability contract address
+     * @dev This function sets the interoperability contract address
+     * @param interop_ The address of the interoperability contract
+     * @return None
+     */
     function setInteroperabilityContract(address interop_) external;
 
     //==============================================================================//
